src/Php55/Php55.php

```diff
@@ -39,9 +39,20 @@ public static function json_last_error_msg()
      */
     public static function hash_pbkdf2($algorithm, $password, $salt, $iterations, $length = 0, $rawOutput = false)
     {
+        // Pre-hash for optimization if password length > hash length
+        $hash_length = \strlen(
+            \hash($algorithm, '', true)
```

Comment:
should be on one line, without the `\` (only strlen has, and a few others, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048 for some background on the topic if you want)